libs/components/modals/src/lib/modules/confirm/confirm.component.ts

```diff
@@ -50,6 +50,8 @@ export class SkyConfirmComponent implements OnDestroy {
   readonly #idSvc = inject(SkyIdService);
 
   constructor() {
+    this.#config.type ??= SkyConfirmType.OK;
```

Comment:
Functionally, the confirm defaults to "OK" when the `type` is not defined. However, the harness derives the type from a CSS class name in the template. Explicitly setting the type to "OK" when it's not defined by the consumer ensures that the appropriate classes are set in the template and the harness returns the correct type.